radio/src/targets/horus/board.cpp

```diff
@@ -175,6 +175,7 @@ void boardInit()
   init2MhzTimer();
   init1msTimer();
   usbInit();
+  usbDeInit();
```

Comment:
I introduced two new functions `usbStart()` and `usbStop()` to make it more clear. a4d899844e6ae5ef0967cae8fc02da562e4ccc1f